package/files/update_rmt_app_dir_permissions.sh

```diff
@@ -16,3 +16,17 @@ if [[ $app_dir_ownership == "_rmt nginx" ]]; then
 
   find -P $app_dir -type f -user _rmt -group nginx | xargs -I {} chown -h root:root {}
 fi
+
+# Change secrets encrypted and key files to nginx readable
+secret_key_files=('config/secrets.yml.key' 'config/secrets.yml.enc')
+
+for secretFile in $secret_key_files; do
+  file_path="$app_dir/$secretFile"
+  if [[ -e $file_path ]]; then
+    if [[ "$(stat -c "%U %G" $file_path)" == "root root" ]]; then
+      chown -h root:nginx $file_path
+      chmod 0640 $file_path
```

Comment:
first chmod, then chown please

Reply:
Due to my not existing knowledge in this regard: What is the problem of doing `chown` and then `chmod`? Probably simple but don't see the tree in the forest, yet :)

Reply:
Done

Reply:
with chown you transfer control of the file to the user. It's better to first have it fully setup, then transfer ownership. It's more of hardening/cosmetic suggestion :)